client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -126,9 +125,9 @@ const BrewRenderer = (props)=>{
 
 	const renderStyle = ()=>{
 		if(!props.style) return;
-		const cleanStyle = props.style; //DOMPurify.sanitize(props.style, purifyConfig);
+		const cleanStyle = safeHTML(`<style>${props.style}</style>`);
```

Comment:
This region of code will need a small rework for User Brews.

Reply:
I believe my thinking here was the `<style>` is a safe tag, so there's no reason to exclude it from the sanitization process. That said, I have no objection to returning to the original configuration of sanitizing `props.style` and then wrapping the string in `<style>` tags at injection time - would that be less of a conflict with the User Brews PR?

Reply:
It's not so much that as it is the User brews rearranged some things. This function is removed.

Whichever applied first, some tweaks will be needed. :)

Reply:
I've just pushed a merge conflict resolution that should solve this now.

Reply:
Oh hey it would probably help if I actually parsed the text through the function that is the whole point of this PR, too...